### DISCUSSION TITLE
Feature Request: NDI Output (via FFMPEG?)

### DISCUSSION THREAD

**Discussion by krisdahl** [OvenMediaLabs/OvenMediaEngine#1195] [Ideas] [answered]
It would be great if there was a workflow to take a source coming into OME and be able to output it as a NDI source on the network.

**Comment by nums** [comment, marked as answer]
Hello @krisdahl , maybe this repo can help : https://framagit.org/tytan652/ffmpeg-ndi-patch ?

To input OME stream on ffmpeg, now it's possible with LLHLS, if you want ultra low latency, it's not possible now. 
You have to wait WHEP implementation in OME and FFMPEG, or SRT output in OME.
